projects/js-packages/components/components/loading-placeholder/style.module.scss

```
@@ -0,0 +1,18 @@
+@keyframes pulse {
+  0% {
+    opacity: .6;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: .6;
+  }
+}
+
+.loadingPlaceholder {
+  background-color: #dcdcde;
```

```suggestion
  background-color: var( --jp-gray );
```

All components use the CSS variables. If it does not work in your application, none of the components will.
To fix that, you can wrap the app container with `<ThemeProvider>`, that injects the CSS variables, like this:
https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/admin/index.js#L50